vhdbuilder/packer/components.json

```diff
@@ -278,8 +278,8 @@
       ],
       "multiArchVersions": [
         "v1.23.0",
-        "v1.23.0.1",
```

Comment:
"v1.23.0.1", not in use?

Reply:
yes, it was skipped when (another) Andy bump to 1.24